modules/client-configuration/pages/supported-features.adoc

```diff
@@ -47,3 +47,29 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
 | {ubuntu} 18.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
 | {ubuntu} 16.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
 |===
+
+
+
+== Supported Tools Packages
+
+The ``spacewalk-utils`` package is an additional package for {productname} that can provide additional services and features.
+
```

Comment:
I think we do not need the historical and version details here.  This info is something for the release notes only.

Reply:
I added that because I was fairly certain we used to have some warnings about "`spacewalk-utils` is not supported" but then I couldn't find it. Happy to take it out, though.